### PR TITLE
updated annotation json schema

### PR DESCRIPTION
**Pull request by sharifX**
Updated JSON schema based on the new RFC. The changes include clarifying the terms and name spaces. Adding FieldValue and ClassValue for selectors. Adding the four motivation types. adding motivatedBy and also AggregateRating.

### CODE REVIEW

**Review by TomDijkema — COMMENTED** [review]
I have a view comments/questions about the new annotation model

File: data-model/annotations/schema/annotation2.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    "as:generator": {
+      "type": "object",
+      "properties": {
+        "ods:type": { "type": "string" },
+        "foaf:name": { "type": "string" },
+        "ods:id": { "type": "string" }
+      },
+      "required": ["ods:type", "foaf:name", "ods:id"]
+    },
+    "dcterms:issued": { "type": "string" },
```
Comment (southeo) [inline data-model/annotations/schema/annotation2.json:86]: I don't see this term in the data model

**Review by southeo — COMMENTED** [review]
Would be really nice to have file this replace the existing annotation.json file instead of having a second one called annotation2.json.

File: data-model/annotations/schema/annotation2.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    "oa:target": {
+      "type": "object",
+      "properties": {
+        "ods:id": { "type": "string" },
+        "ods:selector": {
+          "oneOf": [
+            {
+              "type": "object",
+              "properties": {
+                "ods:type": { "type": "string", "enum": ["ods:FieldValueSelector"] },
```
[the remark this answers is not in the document]
Comment (sharifX) [reply data-model/annotations/schema/annotation2.json]: done

File: data-model/annotations/schema/annotation2.json [hunk 3]
```diff
@@ -0,0 +1,108 @@
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "type": "object",
+  "properties": {
+    "ods:id": { "type": "string" },
+    "ods:type": { "type": "string" },
```
[the remark this answers is not in the document]
Comment (sharifX) [reply data-model/annotations/schema/annotation2.json]: done

File: data-model/annotations/schema/annotation2.json (hunk 1 above)
Comment (sharifX) [reply data-model/annotations/schema/annotation2.json]: updated this to generated. that is what w3c uses but it does not have any namespace attached to it.

**Comment by sharifX** [comment]
> Would be really nice to have file this replace the existing annotation.json file instead of having a second one called annotation2.json.
after the review is done, I will update annotation.json with the approved content. if I do this now, it will create another pull request.

**Pull request merged**